stable/app/templates/deployment.yaml

```
@@ -21,6 +24,10 @@ spec:
         {{- end }}
       labels:
         {{- include "app.selectorLabels" . | nindent 8 }}
+        {{- include "app.labels" . | nindent 8 }}
+        {{- range $key, $value := .Values.labels }}
```

Let's have a separate labels for this called `podLabels` instead?

```suggestion
        {{- range $key, $value := .Values.podLabels }}
```